commons/com.b2international.index.api/src/com/b2international/index/mapping/DocumentMapping.java

```diff
@@ -154,6 +155,13 @@ public Field getField(String name) {
 		return fieldMap.get(name);
 	}
 	
+	public Class<?> getFieldType(String key) {
```

Comment:
[LuceneQueryBuilder#visit(ScoreFunction)](https://github.com/b2ihealthcare/snow-owl/blob/develop/commons/com.b2international.index.lucene/src/com/b2international/index/query/LuceneQueryBuilder.java#L172) would benefit from this shortened method as well.